pkg/dashboard/static/index.html

```diff
@@ -74,7 +74,27 @@
                         style="height: 1.2rem; vertical-align: text-bottom; filter: grayscale(00%);"></a>
             </div>
             <div class="separator-vertical"><span></span></div>
-            <i class="btn bi-power text-muted p-2 m-1 mx-2" title="Shut down the Helm Dashboard application"></i>
+            <!-- <i class="btn bi-power text-muted p-2 m-1 mx-2" title="Shut down the Helm Dashboard application"></i> -->
+            <button type="button" class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#Modal">
```

Comment:
Let's keep the original button, the purpose of the issue were only to add the modal. Attaching the handler to `i.bi-power` would work just fine